yt_dlp/extractor/youtube.py

```diff
@@ -1043,7 +1056,8 @@ def _extract_video(self, renderer):
             url = f'https://www.youtube.com/shorts/{video_id}'
 
         time_text = (self._get_text(renderer, 'publishedTimeText', 'videoInfo')
-                     or self._get_text(reel_header_renderer, 'timestampText') or '')
+                     or self._get_text(reel_header_renderer, 'timestampText')
+                     or self._get_text(renderer, 'publishedTimeText', 'simpleText') or '')
```

Comment:
I don't think this does what you think. The `publishedTimeText` extraction is already handled by the first `get_text` here, so this is redundant?